custom_components/abb_powerone_pvi_sunspec/api.py

```diff
@@ -158,6 +158,7 @@ def init_modbus_data(self):
         self.data["comm_model"] = ""
         self.data["comm_version"] = ""
         self.data["comm_sernum"] = ""
+        self.data["mppt_nr"] = 1
```

Comment:
Another idea: why don't we add "MPPT Modules" as a sensor too?

Reply:
included in last commit